### ISSUE TITLE
Inconsistent feedback style

### ISSUE THREAD

**Issue by aharjula** [maths/moodle-qtype_stack#143]
The div generated from PRT-feedback inside question text is not the same as for the same things in specific feedback.

In particular in render.php do we need "stackprtfeedback" or, could we use "outcome" as a class to be consistent with that used in specific feedback. Do we need "stackprtfeedback" as a class at all? Can we somehow inherit the CSS properties of "outcome".

[Chris is in Finland this week we are talking about this]

-Matti

**Comment by timhunt** [comment]
Sorry I forgot to reply here.

The outcome class is used by the Moodle question system, for a specific thing. In particular it is one of the major divs used to lay out the whole question, so different themes may do lay-out-y things with it that we would not wish to have applied when PRT feedback is mised in with the question text.

I agree it woudl be nice if in CSS you could just say "That the colours and boarder for stackprtfeedback from .outcome", but you can't. You just have to ensure that .stackprtfeedback and .outcome are  styled the same in your theme.

If you use a higher-level tool to generate your CSS (e.g. Less or recesss) then you can do that sort of inheritance.Moodle is starting to do that for some themes, but not others.

So, in summary: there is no simple change that makes things better than they are, although the current situation is not ideal.

**Comment by sangwinc** [comment]
Thanks Tim,
I don't understand enough about these things so don't have anything of substance to add here. Matti, if you're happy please close the issue.

Chris

**Comment by aharjula** [comment]
Oh well so we'll have to do our own CSS modification guide for our partners so that they may use STACK and not complain about this. Unfortunate, but understandable.
